.azure/pipelines/azure-pipelines-external-release.yml

```diff
@@ -1,6 +1,8 @@
 ######################################
-# NOTE: Before running this pipeline to generate a the GitHub Release and new nuget packages, update the VersionPrefix value in Version.props file
-# NOTE: When Version.props file is modified, this pipeline will automatically get triggered
+# NOTE: Before running this pipeline to generate a new nuget package, update the version string in three places
```

Comment:
undo the changes to this file, including newline at the end.

Reply:
Fixed by 3b838b4fd9fcfe785ceeb9a6ae52a36c4ddbfcb5